internal/pkg/model/receiver.go

```diff
@@ -1,5 +1,23 @@
 package model
 
+import (
+	"time"
+
+	"github.com/c2h5oh/datasize"
+)
+
+type ImportCondition struct {
+	Count int               `json:"count" validate:"min=1,max=10000000"`
+	Size  datasize.ByteSize `json:"size" validate:"min=100B,max=50MB"`
```

Comment:
`min=100B,max=50MB` toto funguje? Interne je to `uint64`, tak by sa to malo aj validovat ako cislo:
https://github.com/c2h5oh/datasize/blob/859f65c6625b78dfa569149781d2e753c40a1db5/datasize.go#L10